src/main/play-25/uk/gov/hmrc/hmrcfrontend/views/html/components/package.scala

```diff
@@ -36,10 +37,12 @@ package object components extends Utils with Aliases {
 
   private lazy val accessibilityStatementConfig = Play.current.injector.instanceOf[AccessibilityStatementConfig]
 
-  lazy val hmrcFooterItems = new HmrcFooterItems(accessibilityStatementConfig)
+  lazy val HmrcFooterItems = new HmrcFooterItems(accessibilityStatementConfig)
 
   lazy val HmrcFooter = hmrcFooter
 
+  lazy val HmrcFooterHelper = hmrcFooterHelper
```

Comment:
At the moment, although we have moved the underlying twirl template to a separate package, it is actually exported in the components package with all of the other components.  Although this follows the pattern of play-frontend-govuk's formWithCSRF component, it means we are still going to potentially get namespace clashes when we start introducing 'wrapper' components for the components ported from hmrc-frontend/govuk-frontend. We can properly move the component like this: https://github.com/hmrc/play-frontend-hmrc/compare/PLATUI-752_upgrade-to-1.19.0...PLATUI-752_upgrade-to-1.19.0_split?expand=1

Reply:
I'm also suggesting we rename to hmrcStandardFooter to better reflect what this component does. @JoPintoPaul - what do you think?